packages/shared/src/components/cards/ArticlePostCard.tsx

```diff
@@ -91,7 +91,10 @@ export const ArticlePostCard = forwardRef(function PostCard(
             onMenuClick={(event) => onMenuClick?.(event, post)}
             onReadArticleClick={onReadArticleClick}
           />
-          <CardTitle lineClamp={showFeedback ? 'line-clamp-2' : undefined}>
+          <CardTitle
+            lineClamp={showFeedback ? 'line-clamp-2' : undefined}
+            className={!showFeedback && 'min-h-[4.875rem]'}
```

Comment:
I wanted to use `3lh`, but [firefox does not support](https://caniuse.com/mdn-css_types_length_lh) that. So 4.875rem is 3x1.625rem which is the lineheight for CardTitle.

Reply:
Does the whole card not have a fixed height already?

Reply:
I see now that card has a min-height of `22.5rem`. Going to see if I can increase that by a bit instead

Reply:
I think ideally that's fine, maybe the upvote just adds too much somewhere?

Reply:
Bumped it up to 23.75rem